lib/compile/bundles/bootstrap.js

```diff
@@ -0,0 +1,21 @@
+import most from "most";
+
+import resolveAsset from "../modules/resolve";
+import loadAst from "../modules/load-ast";
+
+export default function (compilation) {
+  return most
+    .from(compilation.opts.emit)
+    .filter(def => def.entry || def.split)
+    .map(function bootstrapBundle (bundleDef) {
```

Comment:
Why is this a named function rather than an arrow function? Also, you required this into `index.js` under the same function name so you will have two stack frames with `bootstrapBundle` if you care.